lib/mli_parser.ml

```diff
@@ -1,7 +1,8 @@
 open! Compat
+module Loc = Odoc_parser.Loc
```

Comment:
I'd rather keep the explicit module name accross the code if you don't mind. I find that it helps not to use such aliases.

If the code is to bloated, I prefer the use of local opens.